pkg/security/resolvers/mount/resolver.go

```diff
@@ -110,28 +110,32 @@ func (mr *Resolver) SyncCache(pid uint32) error {
 	return err
 }
 
-// syncCache update cache with the first working pid
-func (mr *Resolver) syncCache(pids ...uint32) error {
-	var err error
-	var mnts []*mountinfo.Info
+func (mr *Resolver) syncPid(pid uint32) error {
+	mnts, err := kernel.ParseMountInfoFile(int32(pid))
+	if err != nil {
+		return err
+	}
 
-	for _, pid := range pids {
-		mnts, err = kernel.ParseMountInfoFile(int32(pid))
-		if err != nil {
-			mr.cgroupsResolver.DelPID(pid)
```

Comment:
With the changes we're not deleting pids in the cgroup resolver when failing to read the mountinfo file anymore. Is this intended?